src/Crocodealer/Config.hs

```diff
@@ -0,0 +1,17 @@
+module Crocodealer.Config
+       ( Repo(..)
+       , Config(..)
```

Comment:
I usually prefer to add and extra space in between data type and its exposed constructors
```suggestion
       , Config (..)
```